src/cljdoc/render/build_log.clj

```diff
@@ -3,7 +3,9 @@
             [cljdoc.render.layout :as layout]
             [cljdoc.util :as util]
             [cljdoc.util.datetime :as dt]
-            [cljdoc.server.routes :as routes])
+            [cljdoc.server.routes :as routes]
+            [taoensso.nippy :as nippy]
+            [clojure.stacktrace :refer [print-stack-trace]])
```

Comment:
I'd suggest to only use `:refer` for very commonly used symbols as described here: https://stuartsierra.com/2016/clojure-how-to-ns.html